src/main/java/com/google/devtools/build/lib/authandtls/AuthAndTLSOptions.java

```diff
@@ -61,17 +61,32 @@
   )
   public String googleCredentials;
 
+  @Deprecated
   @Option(
     name = "tls_enabled",
     defaultValue = "false",
     documentationCategory = OptionDocumentationCategory.UNCATEGORIZED,
+    deprecationWarning = "Deprecated. Please specify a valid protocol in the URL (https or grpcs).",
     effectTags = {OptionEffectTag.UNKNOWN},
     help =
-        "Specifies whether to use TLS for remote execution/caching and the build event service"
-            + " (BES)."
+        "DEPRECATED. Specifies whether to use TLS for remote execution/caching and "
```

Comment:
Please create an github issue for this incompatible flag and reference it in the help text. The github issue should state in which release this flag will be flipped and how to migrate.

Reply:
done